ansible/collections/bento/common/roles/opensearch_dataloading/tasks/opensearchbackup.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+- name: listing the contents of the workspace after extracting
+  shell: pwd
+  register: shell_result
+
+- debug:
+        var: shell_result.stdout_lines
+
+- name: listing the contents of the workspace after extracting
+  shell: ls
+  register: shell_result
+
+- debug:
+        var: shell_result.stdout_lines
+
```

Comment:
can we comment/remove the debug here prior to merging? I think this is very useful for any testing but will create extra unrequired logs if this is left in the final version of the scripts